docs/extended_api/pipelines/pipeline/producer_acquire.md

```diff
@@ -0,0 +1,26 @@
+---
+grand_parent: Pipelines
+parent: cuda::pipeline
+---
+
+# `cuda::pipeline::producer_acquire`
+
+```c++
+// Header `<cuda/pipeline>`.
```

Comment:
Same comment about "declared in" style as earlier.

This file needs a comment that this functionality is not a part of the github-released open-source library.